pyproject.toml

```
@@ -65,6 +65,73 @@ source = "vcs"
 [tool.hatch.metadata]
 allow-direct-references = true
 
+# ======================================================================================
+# Pixi configuration
+# ======================================================================================
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
+
+[tool.pixi.dependencies]
+python = ">=3.12,<3.13"
```

```suggestion
python = "~=3.13"
```
Just search and replace from here.